src/components/nve-darkmode-switch/nve-darkmode-switch.component.ts

```diff
@@ -0,0 +1,69 @@
+import { INveComponent } from '@interfaces/NveComponent.interface';
```

Comment:
Jeg er litt usikker på hvorfor vi trenger to separate switch knappene, en for lyst tema og en for mørkt tema. Kan de ikke slås sammen og bare settes til 'checked' når brukeren foretrekker mørk modus? Er det noen begrensninger som gjør at vi må ha dem separat som jeg ikke ser?

Reply:
darkmode-switch er for å bytte tema, ikke for å bruke i darkmode. nve-darkmode-switch bruker nve-switch men med mye ekstra funksjonalitet

Reply:
Jeg la inn en link i nve-switch om det